gcn_classic_text_to_json/notices/calet/__init__.py

```diff
@@ -0,0 +1,112 @@
+import email
+import json
+import os
+
+import requests
+
+from ... import conversion, utils
+
+input = {
+    "standard": {
+        "id": "TRIGGER_NUM",
+        "ra": "POINT_RA",
+        "dec": "POINT_DEC",
+        "alert_datetime": "NOTICE_DATE",
+        "trigger_time": ["TRIGGER_DATE", "TRIGGER_TIME"],
+    },
+    "additional": {
+        "url": ("LC_URL", "string"),
+        "rate_snr": ("SIGNIFICANCE", "float"),
+        "foreground_duration": ("FOREGND_DUR", "float"),
+        "background_duration": ("BACKGND_DUR1", "float"),
+    },
+}
+
+
+def text_to_json_calet(notice, record_number):
```

Comment:
This is a bit unusual to me. Typically we don't put logic in __init__.py files. 
It is much more common to see something like the following:
```
from my_package import main_function
main_function()
```
so that would end up with the functions below existing in a separate file and then being imported and called as needed.
```
from file_you_created import create_all_calet_jsons
create_all_calet_jsons()
```

There are a couple of reasons for this:
1. code readability. People typically aren't looking in __init__ files for functions. In cases where the package is larger and there might be a large quantity of functions that go in there, it can get really long and really hard to read. It isn't hard to read here because there is not a lot of code, but it's good to get in the habit of putting things in a standard place.
2. Again, not really an issue here because it's a small amount of code, but because everything in this file gets run at import. So that means if you have a lot of code in the __init__ file, it can eventually lead to slow import times.

Reply:
I was initially trying to configure the way you've described but I think the reason that I formatted the files this way was that I was struggling with figuring out the relatives import and tried a few different things and this format stuck. But now that I have relative imports working I could certainly change it back. I could define a conversion.py file for each subpackage which has most of the logic and keep the core stuff in the __init__.py